frontend/src/lib-customizations/types.d.ts

```diff
@@ -111,6 +111,7 @@ interface BaseFeatureFlags {
    */
   experimental?: {
     ai?: boolean
+    attendanceReservation?: boolean
```

Comment:
I don't think we should use these anymore. Maybe rebase after I merge and then on unit page you can check the backend response in the same way as I did with the "add mobile device" feature.

Reply:
I personally see a difference between feature flags and "pilot units". But is the idea to get rid of frontend feature flags completely?

Reply:
I'm not sure. If the feature is such that it will be released in phases and/or never to some units, then I think it makes sense to design it already to use the db based unit specific model. In those cases the additional global flag seems just redundant and confusing to me. But if the feature is for example for service workers or otherwise released fully in a big bang once it's ready then the frontend feature flag is the way to go.